test/fixtures/forms.yml

```diff
@@ -4,10 +4,10 @@ one:
   name: MyString
   version: 1
   version_independent_id: F-1
-  created_by_id: 1
```

Comment:
While this works, the preferred way is to use the association features of fixtures. You should be able to do this with:

`created_by: not_admin`

See section 3.2.2 http://guides.rubyonrails.org/testing.html#the-low-down-on-fixtures